checklists/lz_checklist.en.json

```diff
@@ -1207,11 +1213,19 @@
     {
       "category": "Security",
       "subcategory": "Secrets",
-      "text": "Use a vault per application per environment per region.",
+      "text": "Don't share Key Vault instances between applications to avoid secret sharing across environments.",

```

Comment:
Did you change this on purpose? I think Charles changed this checklist item over the weekend, you might have an old version of it...